extensions/wikia/Search/templates/WikiaSearch_CrossWiki_result.php

```diff
@@ -14,9 +11,9 @@
 		<p class="description"><?= $description; ?></p>
 
 		<ul class="wiki-statistics subtle">
-			<li><?= $pagesMsg ?></li>
-			<li><?= $imgMsg ?></li>
-			<li><?= $videoMsg ?></li>
+			<li><?= $pages['count'] ?> <?= $pages['message'] ?></li>
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "count" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "message" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)